src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -124,6 +124,8 @@ public String getCustomMessage() {
         return customMessage;
     }
 
+	public SlackNotifier(){};
```

Comment:
Nope, formatting is important and I look for it when I'm merging.  Thanks for raising the issue.  Tabs and trailing spaces aren't coherent with the code style.  Please fix.